lib/rack/pact_broker/invalid_uri_protection.rb

```diff
@@ -12,19 +12,21 @@ module PactBroker
     class InvalidUriProtection
       include ::PactBroker::Messages
 
+      CONSECUTIVE_SLASH = /\/{2,}/
+
       def initialize app
         @app = app
       end
 
       def call env
         if (uri = valid_uri?(env))
           if (error_message = validate(uri))
-            [422, {"Content-Type" => "text/plain"}, [error_message]]
+            [422, headers, [body(env, error_message, "Unprocessable", "validation-errors", 422)]]
```

Comment:
`validation-errors` to me sounds like a business problem, where as this is an invalid HTTP request. I reckon we stick with "unprocessable-entity" like the HTTP status says.

Reply:
Maybe https://problems-registry.smartbear.com/invalid-parameters/ instead?

`unprocessable-entity` doesn't exist in our problem registry

Reply:
Ah, I see what you mean. Let's ask for it. Though, the guidelines do say that for generic http error types, we should use one of the public type Urls. But then, how does the decorator know which base URL to use? It just gets more complicated than it's worth.

Reply:
This is probably best. "invalid path parameter" https://problems-registry.smartbear.com/invalid-request-parameter-value/